includes/admin.form.inc

```diff
@@ -44,6 +44,13 @@ function islandora_usage_stats_admin_form() {
     '#default_value' => variable_get('islandora_usage_stats_exclude_bots', '1'),
   );
 
+  $form['usage']['islandora_usage_stats_exclude_bots_regex'] = array(
+    '#type' => 'textfield',
+    '#title' => t('Bot Exclusion Regex'),
+    '#description' => t('The regular expression used to filter out bots'),
+    '#default_value' => variable_get('islandora_usage_stats_exclude_bots_regex', '/bot|rambler|spider|crawl|slurp|curl|^$/i'),
```

Comment:
The new admin setting is not saving. Add `variable_set('islandora_usage_stats_exclude_bots_regex', $form_state['values']['islandora_usage_stats_exclude_bots_regex']);` to the bottom of `islandora_usage_stats_admin_form_submit()` and it will work.